src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -1005,6 +1005,59 @@
     crafted: true
 """
 
+helps['storage container-rm'] = """
+type: group
+short-summary: Manage Azure containers using the Microsoft.Storage resource provider.
+"""
+
+helps['storage container-rm create'] = """
+type: command
+short-summary: Create a new container under the specified storage account.
+examples:
+  - name: Create a new container under the specified storage account.
+    text: az storage container-rm create --storage-account myAccount --name myContainer
```

Comment:
use something like mystorageaccount for storage account value because only lower case is accepted for storage account